src/fides/api/util/consent_util.py

```diff
@@ -575,62 +624,75 @@ def load_default_notices_on_startup(
 
 def load_default_experience_configs_on_startup(
     db: Session, notice_yaml_file_path: str
-) -> None:
+) -> Tuple[List[ExperienceConfigTemplate], List[PrivacyExperienceConfig]]:
     """
-    On startup, loads default ExperienceConfigs into the database. Creates the defaults
-    if they don't exist, otherwise, updates them with any new values from the default
-    yaml file if applicable.
+    On startup, loads default ExperienceConfig Templates into the database.
+
+    Creates ExperienceConfigs/Translations/Historical records, as well as linking regions (via
+    Privacy Experience) and Notices as long as this ExperienceConfig is new.
     """
     logger.info(
         "Loading default privacy experience configs from {}", notice_yaml_file_path
     )
     with open(load_file([notice_yaml_file_path]), "r", encoding="utf-8") as file:
         experience_configs = yaml.safe_load(file).get("privacy_experience_configs", [])
 
+        template_schemas: List[ExperienceConfigCreateWithId] = []
         for experience_config_data in experience_configs:
-            create_default_experience_config(db, experience_config_data)
+            template_schemas.append(
+                ExperienceConfigCreateWithId(**experience_config_data)
+            )
+
+        privacy_experience_config_templates = (
+            upsert_privacy_experience_config_templates_util(db, template_schemas)
+        )
+
+        # Determine which templates don't have corresponding records in Privacy Experience Config
+        new_templates: List[ExperienceConfigTemplate] = [
+            template
+            for template in privacy_experience_config_templates
+            if template.id
+            not in [
+                origin for (origin,) in db.query(PrivacyExperienceConfig.origin).all()
+            ]
+        ]
```

Comment:
We previously didn't create Experience Config Templates, only notice-level templates. However, as we are shipping with out-of-the-box translations on the plus side, it would be helpful to have these stored in the database. New logic upserts the templates, and then creates Experience Configs only if they are new.  Data migration needs to backfill these templates, and link the origin.